### DISCUSSION TITLE
Ignite function

### DISCUSSION THREAD

**Discussion by TheSilverDude** [wiremod/wire#2589] [Suggestions]
This suggestion is simple:

Add an E2 function; the following one that is:

          e:ignite()

ignite(); when given an entity would be set to 0 by default; and if set to 1 would light the entity on fire. (I'd suggest not making this applicable to players; although I don't even know if it would be possible to use it)

**Comment by Denneisk** [comment]
Would be fitting with that new *damage* extension.
